main.go

```diff
@@ -7,6 +7,6 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	server.RegisterPostProcessor(new(PostProcessor))
+	server.RegisterBuilder(new(Builder))
```

Comment:
Yes, the docs actually say this, but the `ServeBuilder()` function does not seem to be existent. All occurrences I could find were in documentation files. So I checked some other repositories ([here](https://github.com/search?utf8=%E2%9C%93&q=packer-builder&type=) and [here](https://github.com/search?q=packer+post+processor&type=Repositories)) and did the same thing they do.